cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -163,6 +162,13 @@ public static String metricAndLogConfigsName(String cluster) {
         return cluster + METRICS_AND_LOG_CONFIG_SUFFIX;
     }
 
+    /**
+     * Get the name of the TO role binding given the name of the {@code kafkaResourceName}.
+     */
+    public static String roleBindingName(String kafkaResourceName) {
```

Comment:
We are using "cluster" as parameter name in all the other methods. Why changing to "kafka ResourceName"?